src/Plugin_Command.php

```diff
@@ -622,7 +622,7 @@ protected function get_item_list() {
 				'version' => $details['Version'],
 				'update_id' => $file,
 				'title' => $details['Name'],
-				'description' => $details['Description'],
+				'description' => wordwrap( $details['Description'] ),
```

Comment:
Should this be applied when `--format=json`, or only for `--format=table`?

Reply:
Ah, I see that `wp plugin get` does it always...

Reply:
we are using a command with --format=json option